### ISSUE TITLE
[TIL] 오예은_201222

### ISSUE THREAD

**Issue by dsdoris** [codestates/ds-TIL#1236]
## 키워드
`경사하강법` `역전파(backpropagation)` `loss function` `keras`

#

> 고민이 많다.
